tinygrad/runtime/ops_clang.py

```diff
@@ -18,7 +18,7 @@
   'Darwin': {'cflags':'-lm -fPIC --rtlib=compiler-rt ', 'ext':'dylib', 'exp':''}
 }[platform.system()]
 
-CLANG_PROGRAM_HEADER = '#include <math.h>\n#define max(x,y) ((x>y)?x:y)\n#define int64 long\n#define half __fp16\n#define uchar unsigned char\n#define bool uchar\n'
+CLANG_PROGRAM_HEADER = '#include <math.h>\n#define max(x,y) ((x>y)?x:y)\n#define int64 long\n#define half __fp16\n#define uchar unsigned char\n#include <stdbool.h>\n'
```

Comment:
`Tensor([0.1,0.2], dtype=dtypes.float, device="CLANG").cast(dtypes.bool).numpy()`
and generally fp values between 0 and 1 fail on clang because they get truncated